app/packages/core/src/components/Modal/use-modal-selective-rendering.ts

```diff
@@ -54,9 +59,22 @@ export const useImavidModalSelectiveRendering = (
           return;
         }
 
-        const thisFrameId = `${id}-${currentFrameNumber}-${getColoringHash()}`;
+        const thisFrameLutId = `${id}-${currentFrameNumber}`;
+
+        // sync key is not variable between frames, unlike LUT id
```

Comment:
i know there's repetition between modal hook and grid init code, will resolve in follow-up PR.